packages/serialize/src/index.js

```diff
@@ -89,7 +89,8 @@ if (process.env.NODE_ENV !== 'production') {
     }
 
     if (
-      key.charCodeAt(1) !== 45 &&
+      key.charCodeAt(0) !== 64 /* @ */ &&
```

Comment:
Rather than doing this extra check which won’t also work for nested selectors, you could move the oldProcessStyleValue call before this check and check if the value is not equal to an empty string

Reply:
what kind of nested selectors are we talking about?

Reply:
Something like this or any other selector with a hyphen.
```jsx
{
  ':last-of-type': null
}
```

Reply:
oh, right - gonna workaround this soon